frontend/src/employee-frontend/components/unit/unit-reservations/ChildDateModal.tsx

```diff
@@ -143,7 +153,34 @@ export default React.memo(function ChildDateModal({
   const today = LocalDate.todayInHelsinkiTz()
   const editingFuture = date.isAfter(today)
 
-  const absenceOptions = absenceTypes.map((at) => ({
+  const availableBillableAbsences = [...basicAbsenceTypes]
```

Comment:
Should these be done inside `useMemo`? Not a *huge* amount of calculation but it's still done on every render

Reply:
Moved the logic to a function that is called from the initial state function and thus executed only once